src/Core__Date.res

```diff
@@ -1,6 +1,6 @@
 type t = Js.Date.t
 
-type time = float
+type msSinceEpoch = float
```

Comment:
```suggestion
/**
Time, in milliseconds, since the UNIX epoch (January 1, 1970 00:00:00 UTC).
*/
type msSinceEpoch = float
```
cc @glennsl what do you think of this docstring?

Reply:
Perfect! :smiley: